src/config.ts

```diff
@@ -61,11 +61,13 @@ export default {
   SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY || env.SEGMENT_WRITE_KEY,
   NSQ_CIRCUIT_BREAKER_THRESHOLD:
     process.env.NSQ_CIRCUIT_BREAKER_THRESHOLD || env.NSQ_CIRCUIT_BREAKER_THRESHOLD,
-  MAXMIND_GEOLITE2_LICENSE_KEY: process.env.MAXMIND_GEOLITE2_LICENSE_KEY || env.MAXMIND_GEOLITE2_LICENSE_KEY,
```

Comment:
Should retain support for MAXMIND_GEOLITE2_LICENSE_KEY for backwards compatibility. ```GEOIPUPDATE_LICENSE_KEY: process.env.GEOIPUPDATE_LICENSE_KEY || env.GEOIPUPDATE_LICENSE_KEY || process.env.MAXMIND_GEOLITE2_LICENSE_KEY || env.MAXMIND_GEOLITE2_LICENSE_KEY```